### PR TITLE
Made some changes in test file

### PR DESCRIPTION
**Pull request by iamvbenz49**

### CODE REVIEW

File: tests/docs/docs.spec.js [hunk 1]
```diff
@@ -34,10 +34,10 @@ test("docs has gitpod link", async ({ page }) => {
   await expect(page).toHaveURL(/gitpod/);
 });
 
-test('should pass axe wcag accessibility tests', async ({ page }) => {
-  await page.goto('/docs');
+test("should pass axe wcag accessibility tests", async ({ page }) => {
```
Comment (eddiejaoude) [inline tests/docs/docs.spec.js:37]: Thank you for updating to the coding standards

File: tests/events.js [hunk 2]
```diff
@@ -16,9 +16,9 @@ test.fixme("Events listed", async ({ page }) => {
 });
 
 test("should pass axe wcag accessibility tests", async ({ page }) => {
-  await page.goto("/events");
+  await page.goto("/");
```
Comment (eddiejaoude) [inline tests/events.js:19]: This should stay as the `events` page, why the change?

File: tests/docs/docs.spec.js [hunk 3]
```diff
@@ -34,10 +34,10 @@ test("docs has gitpod link", async ({ page }) => {
...
+  await page.goto("/");
```
Comment (eddiejaoude) [inline tests/docs/docs.spec.js:38]: why is the page changing?

File: tests/profile.spec.js [hunk 4]
```diff
@@ -66,3 +66,10 @@ test("should pass axe wcag accessibility tests (_test-wcag-user)", async ({
     .analyze();
   expect(accessibilityScanResults.violations).toEqual([]);
 });
+test("should pass axe wcag accessibility tests", async ({ page }) => {
+  await page.goto("/");
+  const accessibilityScanResults = await new AxeBuilder({ page })
+    .withTags(["wcag2a", "wcag2aa", "wcag21a", "wcag21aa"])
+    .analyze();
+  expect(accessibilityScanResults.violations).toEqual([]);
+});
```
Comment (eddiejaoude) [inline tests/profile.spec.js:75]: this is not required, as the test above already checks the profile page

File: tests/search.spec.js [hunk 5]
```diff
@@ -76,7 +76,7 @@ test.fixme(
 );
 
 test("should pass axe wcag accessibility tests", async ({ page }) => {
-  await page.goto("/search");
+  await page.goto("/");
```
Comment (eddiejaoude) [inline tests/search.spec.js:79]: Not sure why this is no longer testing the search page in the search test file

**Review by eddiejaoude — COMMENTED** [review]
Thank you for your contribution, but I think I will close this and you can submit a new PR that is more focused

**Comment by iamvbenz49** [comment]
I just copy pasted what you gave me in the issue to every test file Isn't that what you told me to do, please explain me further on what to do ?

**Comment by eddiejaoude** [comment]
I gave an example of an accessibility test to add to files that didn't already have it, but you need to adjust the variable path for each page to make sense, otherwise each test is testing the same page not the page for that specific test

**Comment by iamvbenz49** [comment]
Can you be more specific, i didn't get it

**Comment by eddiejaoude** [comment]
I have converted the issue to an EPIC, so that people can do a PR for individual pages, have a look at the discussion in the issue and keep an eye on the related PRs. Also ask questions in the EddieHub Discord, people are there to help

**Comment by eddiejaoude** [comment]
@iamvbenz49 here is a `good first issue` to get you started, let me know in the issue if you would like me to assign it to you https://github.com/EddieHubCommunity/LinkFree/issues/6008
